packages/next/src/server/dev/turbopack-utils.ts

```diff
@@ -61,8 +61,33 @@ export function isWellKnownError(issue: Issue): boolean {
   return false
 }
 
-/// Print out an issue to the console which should not block
-/// the build by throwing out or blocking error overlay.
+/**
+ * Convert given issues into formatted strings if it's non-fatal error (warning).
+ *
+ * If error is warning severity will log to console. Or if error is well known,
+ * log as error to console.
+ */
+export function collectFormattedIssues(issues: Array<Issue>) {
+  const ret = []
+
+  for (const issue of issues) {
+    const formattedIssue = formatIssue(issue)
+    if (issue.severity === 'warning') {
+      printNonFatalIssue(issue)
+      continue
+    } else if (isWellKnownError(issue)) {
```

Comment:
Why only for well know errors? Why don't we show all error on the console?

Reply:
This mimics existing webpack behavior from its plugin https://github.com/vercel/next.js/blob/ff1db77f1bd1298c68debace25ff7062bfcefcda/packages/next/src/build/webpack/plugins/wellknown-errors-plugin/index.ts#L6

That we only emits specific errors into console.

Reply:
I don't see how this plugin does that... It replaces some errors with nicer ones, but keeps all other untouched

Reply:
huh, seems confused between this : https://github.com/vercel/next.js/blob/ff1db77f1bd1298c68debace25ff7062bfcefcda/packages/next/src/build/webpack/plugins/wellknown-errors-plugin/index.ts#L17

so you're thinking to emit any errors into console would be better?

Reply:
yes